### PR TITLE
 test: rename basic story into default

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- all story titles start are called using non-capital letter;
- rename all `basic` story to the `default`/`default story` one;
- replace all imports in the `mdx` files.

### Current behaviour

We need to amend all story names to be:
- all story starts from the non-capital letter;
- rename all `basic` story to the `default` one;
- replace all imports in the `mdx` files.

### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### Testing instructions

Check that CI passes and all imports work properly

### CODE REVIEW

File: src/components/dialog-full-screen/dialog-full-screen.stories.js [hunk 1]
```diff
@@ -123,3 +123,21 @@ export const Nested = () => {
...
+  name: "default",
+  parameters: {
+    chromatic: {
+      disable: true,
+    },
+  },
+};
+
+Nested.story = {
+  name: "default",
```
Comment (mkrds) [inline src/components/dialog-full-screen/dialog-full-screen.stories.js:137]: Two stories with `name: "default"`

File: src/components/popover-container/popover-container.stories.mdx [hunk 2]
```diff
@@ -21,12 +21,12 @@ import PopoverContainer from "carbon-react/lib/components/popover-container";
...
 To use `PopoverContainer`, import the `PopoverContainer` and pass the content as children.
 <Preview>
-  <Story name="basic" parameters={{ info: { disable: true }}} >
+  <Story name="default" parameters={{ info: { disable: true }}} >
     <div style={{height: 100}}>
       <PopoverContainer>
       Contents
       </PopoverContainer>
-      
+
```
Comment (mkrds) [inline src/components/popover-container/popover-container.stories.mdx:29]: This might break that story

Comment (DlgSHi) [reply src/components/popover-container/popover-container.stories.mdx:29]: it was like that before :) it actually breaks the story without those spaces :)

File: src/components/dialog-full-screen/dialog-full-screen.stories.js (hunk 1 above)
Comment (DlgSHi) [reply src/components/dialog-full-screen/dialog-full-screen.stories.js]: fixed

**Review by mkrds — DISMISSED** [review]

File: cypress/features/regression/accessibility/accessibilityDesignSystem.feature [hunk 3]
```diff
@@ -26,8 +26,8 @@ Feature: Accessibility tests - Design System folder
     Then "Advanced Color Picker" component has no accessibility violations
 
   @accessibility
-  Scenario Outline: Component <component> basic default page
-    When I open "Test <component>" component page "basic" in no iframe
+  Scenario Outline: Component <component> default default page
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/accessibility/accessibilityDesignSystem.feature]: fixed

**Review by mkrds — APPROVED** [review]

**Review by KatarzynaQA — APPROVED** [review]

**Pull request merged**